src/__init__.py

```diff
@@ -0,0 +1,32 @@
+sys.path.append("..")
+
+
+sys.path.append("rhos")
+sys.path.append("../rhos")
```

Comment:
`sys.path.append`s should not be needed with the new package structure; please remove them wherever they are present (I won't point out all of them) and verify that things still work.

Reply:
I should have removed them everywhere now.

Reply:
Have another look; there's still at least one left that I've spotted (in `src/__init__.py`)

Reply:
Sorry for this one: I must have made some confusion while editing. I have checked now and removed all the sys.path.append!